datasets/thaisum/thaisum.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+csv.field_size_limit(int(1e6))  # to accommodate large fields
+
+
+_CITATION = """\
+@mastersthesis{chumpolsathien_2020,
+    title={Using Knowledge Distillation from Keyword Extraction to Improve the Informativeness of Neural Cross-lingual Summarization},
+    author={Chumpolsathien, Nakhun},
+    year={2020},
+    school={Beijing Institute of Technology}
+"""
+
+_DESCRIPTION = """\
+ThaiSum is a large-scale corpus for Thai text summarization obtained from several online news websites namely Thairath,
+ThaiPBS, Prachathai, and The Standard. This dataset consists of over 350,000 article and summary pairs
+written by journalists.
+"""
+
+
+class ThaiSumConfig(datasets.BuilderConfig):
+    """BuilderConfig for ThaiSum."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for ThaiSum.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(ThaiSumConfig, self).__init__(**kwargs)
+
+
+class ThaiSum(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Thaisum(datasets.GeneratorBasedBuilder):
```
this needs to be the camel case version of the dataset script name `thaisum`